h-include.js

```diff
@@ -42,7 +42,7 @@ var hinclude;
     classprefix: "include_",
 
     show_content: function (element, req) {
-      var i, include, message, fragment = element.getAttribute('fragment');
+      var i, include, message, fragment = element.getAttribute('fragment') || 'body';
       if (req.status === 200 || req.status === 304) {
         var container = document.implementation.createHTMLDocument().documentElement;
```

Comment:
Just discovered that this throws in IE9 & IE10 - the title argument to `createHTMLDocument()` is not optional.